frontend/src/pages/video/common/comment-block/config.ts

```diff
@@ -0,0 +1,25 @@
+import { CommentMenuOptions, IconName } from 'common/enums/enums';
+
+const matchMenuOptionWithIconName: Record<CommentMenuOptions, IconName> = {
+  [CommentMenuOptions.EDIT]: IconName.EDIT,
+  [CommentMenuOptions.DELETE]: IconName.DELETE,
+  // [CommentMenuOptions.REPORT]: IconName.REPORT,
+};
+
+const matchMenuOptionWithText: Record<CommentMenuOptions, string> = {
+  [CommentMenuOptions.EDIT]: 'Edit',
+  [CommentMenuOptions.DELETE]: 'Delete',
+  // [CommentMenuOptions.REPORT]: 'Report',
```

Comment:
is it should be used in future or you just forget remove this?

Reply:
@WanderningMaster it will be used in future.
I commented it, only for now.